workers/generic-worker/taskcluster_proxy.go

```diff
@@ -57,9 +60,19 @@ func (l *TaskclusterProxyTask) RequiredScopes() scopes.Required {
 }
 
 func (l *TaskclusterProxyTask) Start() *CommandExecutionError {
+	if l.task.D2GInfo != nil {
+		out, err := exec.Command("docker", "network", "inspect", "bridge", "--format", "{{ index (index .IPAM.Config 0 ) \"Gateway\"}}").Output()
+		if err != nil {
+			return executionError(internalError, errored, fmt.Errorf("could not determine docker bridge IP address: %s", err))
+		}
+		l.taskclusterProxyAddress = strings.TrimSpace(string(out))
```

Comment:
This can be simplified/standardized a bit by using our `host.CombinedOutput()` helper, and you'd be able to print from standard error too in your error block.

```suggestion
		out, err := host.CombinedOutput("docker", "network", "inspect", "bridge", "--format", "{{ index (index .IPAM.Config 0 ) \"Gateway\"}}")
		if err != nil {
			return executionError(internalError, errored, fmt.Errorf("could not determine docker bridge IP address. Output: %s. Error: %v", out, err))
		}
		l.taskclusterProxyAddress = strings.TrimSpace(out)
```

Reply:
Curious, why don't you want stderr?

Reply:
The intent here is to capture the ip address on stdout.  If docker happens to print other stuff on stderr for whatever reason, mixing that in with stdout isn't going to end well?